maat-court-data-api/src/main/java/gov/uk/courtdata/repository/RepOrderMvoRepository.java

```diff
@@ -0,0 +1,10 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.RepOrderMvoEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface RepOrderMvoRepository extends JpaRepository<RepOrderMvoEntity, Integer>, JpaSpecificationExecutor<RepOrderMvoEntity> {
```

Comment:
The `JpaSpecificationExecutor` can be removed here, it doesn't seem to be doing anything.

Reply:
This is the standard Repo code which is Auto generated. I would rather leave it as it is, as we may use these features going forward. Similar to generating the Entity class with all the fields as a standard practice